docs/usage-manual/(exported from wiki) Message Passing.txt

```diff
@@ -168,7 +168,7 @@ can call a block's gr::basic_block::_post method directly and pass it a
 message. So any block with an input message port can receive messages
 from the outside in this way.
 
-The following example uses a gr::blocks::pdu_to_tagged_stream block
```

Comment:
@777arc @duggabe - I'm not too familiar with the "exported from wiki" procedure.  Is this file something that should be manually edited until we update the export?

Reply:
grep caught this but I left it as its an export. i think i will have a non-trivial amount of wiki updating to do when this is merged.